dkg-gadget/src/async_protocols/sign/handler.rs

```diff
@@ -62,6 +62,9 @@ where
 				reason: "execute called twice with the same AsyncProtocol Parameters".to_string(),
 			})?;
 
+		let logger0 = params.logger.clone();
+		let logger2 = params.logger.clone();
```

Comment:
What's the point of these names? Why `logger0` and `logger2` and why not `logger1`, where's the consistency? These loggers do different things?

Reply:
Probably because I deleted `logger1`, and didn't translate `logger2` to `logger1` as I was focused elsewhere in the code.